pcdsdevices/mv_interface.py

```diff
@@ -192,9 +191,12 @@ def mv_ginput(self, timeout=None):
         recently active plot. If there are no existing plots, an empty plot
         will be created with the motor's limits as the range.
         """
+        # This is usually bad, but importing this at module level forces
+        # matplotlib to select a backend on import, breaking some things
+        import matplotlib.pyplot as plt  # NOQA
```

Comment:
At times, importing like this is the best option you have. 👍 

(I'm not sure this warrants the 2-line disclaimer above, but that's just my opinion.)

Reply:
Sure, I'll make it a 1-line disclaimer. I just don't want someone to move it back because it looks wrong if you don't know.